packages/picker/src/Picker.ts

```diff
@@ -393,7 +393,7 @@ export class PickerBase extends SizedMixin(Focusable, { noDefaultSize: true }) {
                 .offset=${0}
                 ?open=${this.open}
                 .placement=${this.isMobile.matches ? undefined : this.placement}
-                type="auto"
```

Comment:
Can we bind this as a property if it is going to be dynamic?

Reply:
Yes sure! I can add in OverlayTypes from Overlay and create a property in Picker